### ISSUE TITLE
egon.data.subprocess fails with Python3.6

### ISSUE THREAD

**Issue by ClaraBuettner** [openego/eGon-data#104]
The tasks 'import-zensus-population' and 'import-osm' on the current dev branch fail with the same error caused by egon.data.subprocess: 
```
[2021-01-18 15:31:20,038] {taskinstance.py:1150} ERROR - __init__() got an unexpected keyword argument 'capture_output'
Traceback (most recent call last):
  File "/home/clara/egon_data_dev/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/clara/egon_data_dev/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/clara/egon_data_dev/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/clara/GitHub/eGon-data/src/egon/data/importing/openstreetmap/__init__.py", line 77, in to_postgres
    cwd=os.path.dirname(__file__),
  File "/home/clara/GitHub/eGon-data/src/egon/data/subprocess.py", line 59, in run
    subprocess.run(*args, **kwargs)
  File "/usr/lib/python3.6/subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
TypeError: __init__() got an unexpected keyword argument 'capture_output'
```
@EvaWie and I tested this, we both use python3.6. [This post](https://stackoverflow.com/questions/53209127/subprocess-unexpected-keyword-argument-capture-output/53209196) says that 'capture_output' was added in python3.7 and I can't use it with python3.6.

**Comment by ClaraBuettner** [comment]
I preliminary fixed it here: https://github.com/openego/eGon-data/commit/47679a7356f2dbb386e7de5e868d52cdcbf9c413
But since I just deleted the argument  'capture_output' when the python version is < '3.7', I don't think this is the best way.

**Comment by gnn** [comment]
Your entirely correct. I overlooked the fact that the `capture_output` was added in Python version 3.7. I could work around this limitation by manually implementing what the `capture_output` parameter does, but before I do this, I'd like to ask whether it's possible to drop support for Python versions below 3.7. There are other quality of life improvements in Python 3.7 which I wanted to use, so I've already asked for dropping support for Python versions below 3.7 in issue #59 but didn't get too many answers. Would using Python 3.7 or later be a possibility for you?
